module/VuFind/src/VuFind/ILS/Driver/KohaILSDI.php

```diff
@@ -604,6 +604,22 @@ public function placeHold($holdDetails)
         $this->debug("Needed before date: " . $needed_before_date);
         $this->debug("Level: " . $level);
 
+        // The following check is mainly required for certain old buggy Koha versions
+        // that allowed multiple holds from the same user to the same item
+        $sql = "select count(*) as RCOUNT from reserves where borrowernumber = :rid "
+            . "and biblionumber = :bid";
```

Comment:
You have a point there! I didn't think of multi-volume sets. That would indeed be a case when one might want to place holds on more than one items of a title.
I'll loosen the check and recommit.